checkbox.component.ts

```diff
@@ -0,0 +1,61 @@
+import {Component, EventEmitter, OnInit, Output} from '@angular/core';
+import {FieldConfig} from "../../../interfaces/field.interface";
+import {FormGroup} from "@angular/forms";
+import {SubclassComponent} from "../../subcomponents/subclass.component";

```

Comment:
import order
//System imports

//App imports
```suggestion
import { Component, EventEmitter, OnInit, Output } from '@angular/core';
import { FormGroup } from '@angular/forms';

import { FieldConfig } from '../../../interfaces/field.interface';
import { SubclassComponent } from '../../subcomponents/subclass.component';

```